crates/sui-core/src/authority/authority_store_types.rs

```diff
@@ -105,118 +100,16 @@ pub struct StoreObjectValue {
 pub enum StoreData {
     Move(MoveObject),
     Package(MovePackage),
-    IndirectObject(IndirectObjectMetadata),
+    IndirectObjectDeprecated,
```

Comment:
Are we really that sure that no one has ever enabled this? Its quite scary because the bug won't surface until we actually try to load an indirect object. Obviously we will not be able to handle that case no matter what, but it would be better to get an explicit panic rather than a deserialization failure which likely won't even be logged correctly.

Reply:
we're not 100% sure, but it's very unlikely. The flag was not even documented. Affected FNs(if any) can always restore from a snapshot